lib/utils/style-settings.js

```diff
@@ -39,7 +39,7 @@ function checkStyleSettings (filePath) {
     projectPath = projectPaths.find((p) => absoluteActivePanePath.indexOf(fs.realpathSync(p)) >= 0)
   } catch (e) {
     console.error('Could not get project path', e)
-    return
+    return {}
```

Comment:
You could also write `return settings` like all the other return statements, because settings is an empty object anyways